assets/stylesheets/purple/main.scss

```diff
@@ -18,12 +18,11 @@ body {
   height: 100%;
   text-align: center;
   padding-top: 60px;
-  .content {
+  .content,
+  .content > a {
     margin-bottom: 90px;
-    > a {
-      color: $gray-light;
-      color: fade-out(white, .2);
-    }
+    color: $gray-light;
+    color: fade-out(#ffffff, 0.2);
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Property `color` already defined on line 24